packages/plugin-infra/src/types.ts

```diff
@@ -307,11 +307,28 @@ export namespace Plugin.SNSAdaptor {
         /**
          * Render entry component
          */
-        RenderEntryComponent: (props: { disabled: boolean }) => JSX.Element
+        RenderEntryComponent?: (props: { disabled: boolean; AppIcon: JSX.Element }) => JSX.Element | null
         /**
          * Used to order the applications on the board
          */
-        defaultSortingPriority: number
+        appBoardSortingDefaultPriority?: number
+
+        /**
+         * Used to order the applications on the market list
+         */
+        marketListSortingPriority?: number
+
+        AppIcon: JSX.Element
+
+        name: string
+
+        description?: string
```

Comment:
Please use I18nField for these 2 fields, this allows those strings to be translated.